frontend/__tests__/unit/HomeLoggedOut.test.jsx

```diff
@@ -0,0 +1,59 @@
+import { render, screen } from "@testing-library/react";
+import HomeLoggedOut from "../../src/components/HomeLoggedOut";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import { act } from "react-dom/test-utils";
+
+var axios = require("axios");
+var MockAdapter = require("axios-mock-adapter");
+var mock = new MockAdapter(axios);
```

Comment:
@dec0dOS if I understood correctly what you mean about ES6 and CommonJS, these are the lines you are talking about in the comments on the pull request? If so, I agree with you. After I got it all working, I sort of put this aside and moved on. I can revisit these lines and figure out how to change them to ES6 code instead.